lib/chopperbot/split.ex

```diff
@@ -1,141 +1,36 @@
 defmodule Chopperbot.Split do
-  @moduledoc """
-
-  TODO:
-  [ ] add support for LINE format
-  """
-
-  alias Chopperbot.Split.{Order, Parser}
-
-  @type orders :: list(Order.t())
+  alias Chopperbot.Split.{Order, OrderCalculator, Parser}
 
   @doc """
   Process text input for /split to result
 
   ## Examples
       iex> run("a 100 a 200 b 300 +v +s")
-      {:ok, "a: 353.10 THB\\nb: 353.10 THB\\n---\\n*total: 706.20 THB*"}
+      {:ok, [{"a", 353.1}, {"b", 353.1}, {"_total", 706.2}]}
 
       iex> run("a 1100 b 300 share 200 +s")
-      {:ok, "a: 1,320.00 THB\\nb: 440.00 THB\\n---\\n*total: 1,760.00 THB*"}
+      {:ok, [{"a", 1320.0}, {"b", 440.00000000000006}, {"_total", 1760.0}]}
 
       iex> run("a 1100 b 300 share 200 +invalid -haha")
       {:error, "invalid options: +invalid, -haha"}
 
       iex> run("a 1100 b 300 share five dollars")
       {:error, "invalid inputs: five, dollars"}
   """
-  @spec run(String.t()) :: {:ok, String.t()} | {:error, String.t()}
+  @spec run(String.t()) :: {:ok, [Order.t()]} | {:error, String.t()}
```

Comment:
Now Split.run/1 will return a list of orders instead of plain text, so that it will be more flexible for us to do the text format based on each platform.

Reply:
Nice!